src/comment-upserter.ts

```diff
@@ -22,8 +20,14 @@ export interface CommentUpserter {
    * @param repo - repository that the pull request is in
    * @param pullNumber - number that identifies the pull request
    * @param rules - mention rules to use in the comment
+   * @param commentContent - comment content to print above matching rules table
```

Comment:
This should be `commentConfiguration` instead of `commentContent`, and the parameter description should be updated as it is no longer just the epilogue (same comment applies to line 84)